docs/guides/customizing-components.stories.mdx

```diff
@@ -5,39 +5,40 @@ import { Banner } from "@jobber/components/Banner";
 
 # Customizing components
 
-### Philosophy: easy to use, flexible to customize
+## Balancing simplicity, flexibility, and consistency
 
 Atlantis is designed to make building consistent, accessible, and visually
-appealing user interfaces as effortless as possible. Our goal is to create
-components that work out of the box, requiring minimal setup or configuration
-from consumers.
+appealing user interfaces as effortless as possible for you.
 
-While we strive to cover the most common design scenarios with thoughtful
-defaults, we understand that unique use cases may require customization. For
-these scenarios, we provide mechanisms to extend or modify our components in a
-way that maintains flexibility without compromising the integrity of the
-components.
+We strive to speed up building the most common use cases "out of the box" with
+thoughtful defaults that create consistency for users. When unique scenarios
+arise that may require customization, we provide flexible mechanisms to extend
+or modify our components without compromising the integrity of their most common
+use cases.
 
-We recommend using the default implementations wherever possible to benefit from
-consistency, maintainability, and built in accessibility. However, if you choose
-to customize, please note that the responsibility for ensuring proper
-functionality and cohesion will rest with you.
+We recommend using the default implementations wherever possible to benefit
+from:
 
-Atlantis aims to strike a balance between simplicity, flexibility, and
-consistency.
+- consistency
+- maintainability
+- built-in accessibility
```

Comment:
Open to rolling this back into one larger paragraph, but felt like flagging what you're giving up/taking on by customizing in bullets was useful.